agent/build.gradle

```diff
@@ -0,0 +1,36 @@
+
+plugins {
+    id 'com.github.johnrengelman.shadow' version '1.2.1'
+}
+
+apply from: "$buildScriptsDir/common-java.gradle"
+apply from: "$buildScriptsDir/publishing.gradle"
+
+shadowJar {
+    classifier = ''
+    relocate 'org.objectweb.asm', 'com.microsoft.applicationinsights.agent.dependencies.asm'
+}
+archivesBaseName = 'applicationinsights-agent'
+
+jar {
+    dependsOn shadowJar
+    enabled = false
+    manifest {
+        attributes("Agent-Class": "com.microsoft.applicationinsights.agent.internal.agent.AgentImplementation",
+                "Premain-Class": "com.microsoft.applicationinsights.agent.internal.agent.AgentImplementation",
+                "Can-Redefine-Classes": "true"
+        )
+    }
+}
+
+dependencies {
+    compile group: 'org.ow2.asm', name: 'asm', version: '5.0.3'
+    compile group: 'org.ow2.asm', name: 'asm-commons', version: '5.0.3'
+    testCompile group: 'commons-io', name: 'commons-io', version: '2.4'
+    testCompile group: 'junit', name: 'junit', version: '4.11'
+    testCompile group: 'org.mockito', name: 'mockito-all', version: '1.8.0'
+}
+
+projectPomName = project.msftAppInsights + " Java SDK Agent Module"
```

Comment:
We need to decide how to publish, that is a temporary solution that lets us build and test the agent with the sdk jars